distribution/bwc/build.gradle

```diff
@@ -54,6 +54,16 @@ subprojects {
 
   final String remote = System.getProperty("tests.bwc.remote", "elastic")
 
+  final boolean gitFetchLatest
+  final String gitFetchLatestProperty = System.getProperty("tests.bwc.git.fetch.latest", "true")
```

Comment:
nit: I find these ultra.long.uses.of.dot.separation annoying. I use dots in other system properties within the build, but I try to only use them as grouping, not as an alternative to camelCase. In this case, I think a single word after `tests.bwc.` (ie the "group" of sysprops pertaining to bwc tests) would be sufficient.